scripts/variantstore/wdl/ImportGenomes.wdl

```diff
@@ -185,3 +228,131 @@ task CreateImportTsvs {
   }
 }
 
+# Creates all the tables necessary for the LoadData operation
+# As an optimization, I also generate a (table, dir, files) csv file which contains
+# most of inputs necessary for the following LoadTable task.
+task CreateTables {
```

Comment:
It seems like this is doing a bunch more than creating the tables, it's also preparing a TSV (regardless of if it made the tables).  Why is that an optimization?  If feels like it adds coupling that's not totally obvious